### DISCUSSION TITLE
In-Network Rate File (plan_id required or not required)?

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#7] [Q&A] [answered]
It was my understanding from the rule, that the HIOS ID (14 byte/5 byte) or EIN would be a required field in all three files, yet the schema in the In-Network Rate File has the plan_id as not required.  
It is showing required in the Allowed Amount File.  
Is there an issue with the schema for the In-Network Rate File, plan_id or is it really not required for the In-Network Rate File?

**Comment by shaselton-usds** [comment, marked as answer]
Thanks for the catch. Updates can be found here: https://github.com/CMSgov/price-transparency-guide/commit/cd21a067e63520b2a72add9f2f50afe0b87f44c0
